.github/workflows/svc-bip-api-integration-test.yml

```diff
@@ -49,9 +49,7 @@ jobs:
 
           ./gradlew -p mocks :dockerComposeUp
 
-          ./gradlew :domain-xample:dockerComposeUp
           ./gradlew :app:dockerComposeUp
```

Comment:
`./gradlew :app:dockerComposeUp` this brings up the `svc-bip-api` and i believe that it might be stealing the responses from the integration tests calls or preventing the calls from being made to the mock service at all. When i run through this locally, it fails all the tests. But if i remove this line, the integration tests pass.